runtime/vm/ClassInitialization.cpp

```diff
@@ -157,12 +157,11 @@ performVerification(J9VMThread *currentThread, J9Class *clazz)
 		/* See if this class should be verified:
 		 *
 		 * - Do not verify any class created by sun.misc.Unsafe
-		 * - Do not verify hidden classes
 		 * - Do not verify any class which is marked for exclusion in the optional flags
 		 * - Verify every class whose bytecodes have been modified
 		 * - Do not verify bootstrap classes if the appropriate runtime flag is set
 		 */
-		if (((!J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz)) && !J9ROMCLASS_IS_HIDDEN(romClass))
+		if ((!J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz))
```

Comment:
Isn't this already covered by the `J9CLASS_IS_EXEMPT_FROM_VALIDATION` macro?
```

#define J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz) \
	((J9ROMCLASS_IS_UNSAFE((clazz)->romClass) && !J9ROMCLASS_IS_HIDDEN((clazz)->romClass)) || (J9_ARE_ANY_BITS_SET((clazz)->classFlags, J9ClassIsExemptFromValidation)))
```

Reply:
For hidden class, this macro returns FALSE.  so 
The original code `!J9CLASS_IS_EXEMPT_FROM_VALIDATION(clazz) && !J9ROMCLASS_IS_HIDDEN(romClass)`
is `TRUE && FLASE = FALSE`.

Removing `&& !J9ROMCLASS_IS_HIDDEN(romClass)`, this condition becomes TRUE.